MSAL/src/MSALAccount+Internal.h

```diff
@@ -67,12 +64,17 @@
          homeAccountId:(NSString *)homeAccountId
         localAccountId:(NSString *)localAccountId
            environment:(NSString *)environment
-              tenantId:(NSString *)tenantId;
+              tenantId:(NSString *)tenantId
+         idTokenClaims:(MSIDIdTokenClaims *)idTokenClaims;
```

Comment:
Should we deny using 'new' & 'init'?

Reply:
Since it is not a public API, i may want to keep the new & init to give some flexibility to our uint tests, what do you think? :)